Ports/libfts/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='libfts'
```

Comment:
Should we maybe just implement this ourselves? We have quite a few BSD functions floating around already.

Reply:
I thought about this but fts is not only non-standard but also very rarely used these days. On Alpine Linux, for instance, only [15 packages](https://pkgs.alpinelinux.org/package/v3.18/main/x86_64/musl-fts) depend on fts. I think the musl maintainer made the right call not to implement it but to defer this to a separate library, which programs can link against if they really need fts. IMHO it's not worth bloating Serenity's libc with non-standard obscure functions.

Reply:
Fine by me.